tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -309,6 +312,60 @@ def test_anomaly_detection_alert_seer_timeout_max_retry(self, mock_logger, mock_
         assert resp.data["detail"] == "Timeout error when hitting Seer store data endpoint"
         assert mock_seer_request.call_count == 1
 
+    @with_feature("organizations:anomaly-detection-alerts")
+    @with_feature("organizations:incidents")
+    @patch(
+        "sentry.seer.anomaly_detection.store_data.seer_anomaly_detection_connection_pool.urlopen"
+    )
+    def test_anomaly_detection_alert_not_enough_data(self, mock_seer_request):
+        data = {
+            **self.alert_rule_dict,
+            "detection_type": AlertRuleDetectionType.DYNAMIC,
+            "sensitivity": AlertRuleSensitivity.LOW,
+            "seasonality": AlertRuleSeasonality.AUTO,
+        }
+        mock_seer_request.return_value = HTTPResponse(status=202)
```

Comment:
chatted in person - but i think we actually want this to be a mocked 200
the helper method that this is being pulled through is actually modifying the response which isn't ideal, but for now we should test the existing implementation.

lets add a note here to why we're mocking the one status and asserting the other though